Bottom: 2db3ecfe88c64a5860045bcd32214ce536ba35ab
Top:    c3730d4d38c1401aef816bd37b4262f00eecfcce
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-21 00:11:01 +0100

Refresh of hid-common-draw-the-hole-outli

---

diff --git a/src/crosshair.c b/src/crosshair.c
index 3d352b6..d86a194 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -94,7 +94,7 @@ thindraw_moved_pv (PinType *pv, int x, int y)
   moved_pv.X += x;
   moved_pv.Y += y;
 
-  gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &moved_pv, false, false);
+  gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &moved_pv, true, false);
 }
 
 /* ---------------------------------------------------------------------------
@@ -574,7 +574,7 @@ DrawAttached (void)
         via.Flags = NoFlags ();
         TOGGLE_FLAG (SQUAREFLAG, &via);
 
-        gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &via, false, false);
+        gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &via, true, false);
 
         if (TEST_FLAG (SHOWDRCFLAG, PCB))
           {
